qiskit/providers/aer/noise/passes/relaxation_noise_pass.py

```diff
@@ -0,0 +1,93 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Thermal relaxation noise pass.
+"""
+import warnings
+from typing import Optional, Union, Sequence, List
+
+import numpy as np
+
+from qiskit.circuit import Instruction, QuantumCircuit
+from .local_noise_pass import LocalNoisePass
+from ..errors.standard_errors import thermal_relaxation_error
+
+
+class RelaxationNoisePass(LocalNoisePass):
+    """Add duration dependent thermal relaxation noise after instructions."""
+
+    def __init__(
+            self,
+            t1s: List[float],
+            t2s: List[float],
+            dt: float,
```

Comment:
General question: Are delay units in scheduled circuits always in terms of a backends dt?